### PR TITLE
Feature/heartbeat again

### PR DESCRIPTION
**Pull request by minsulee2**
- Add heartbeat in the simplest way via socket
- More update will be next up when some jobs the tracker now doing is transplanted into peer nodes.
- Now, heartbeat just ping and get pong when the corresponding peer is alive.

### CODE REVIEW

File: p2p/index.js [hunk 1]
```diff
@@ -30,9 +30,7 @@ class P2pClient {
         this, node, minProtocolVersion, maxProtocolVersion, this.maxInbound);
     this.trackerWebSocket = null;
     this.outbound = {};
-    // XXX(minsu): The comment out will be revoked when next heartbeat updates.
-    // this.isAlive = true;
-    // this.heartbeat();   // XXX(minsu): it won't run before updating p2p network.
+    this.heartbeat();
```
Comment (platfowner) [inline p2p/index.js:33]: Suggestion: What about setIntervalHeartbeat()?
Or simply startHeartbeat() / stopHeartbeat()?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: p2p/index.js (hunk 1 above)
Comment (minsulee2) [reply p2p/index.js]: I took startHeartbeat() and stopHeartbeat()! Thanks!

**Comment by minsulee2** [comment]
All addressed. I am merging now.

**Pull request merged**